src/modules/txbuilder/RadixTransactionBuilder.ts

```diff
@@ -35,14 +34,17 @@ import {
     RadixFixedSupplyTokenDefinitionParticle,
 } from '../atommodel'
 
-import { logger } from '../common/RadixLogger'
 import { RadixTokenDefinition, RadixTokenSupplyType } from '../token/RadixTokenDefinition'
+import { AtomOperation, AccountState, createInitialState, LedgerState } from '../account/types'
+import { TransferState } from '../account/RadixTransferAccountSystem'
+import { TokenDefinitionState } from '../account/RadixTokenDefinitionAccountSystem'
 
 export default class RadixTransactionBuilder {
     private BNZERO: BN = new BN(0)
-    private DCZERO: Decimal = new Decimal(0)
 
     private particleGroups: RadixParticleGroup[] = []
+    private actions: Function[] = []
```

Comment:
Are there no other requirements for actions type wise other than being a function?